tests/unit/ops/test_ops_schema.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+
+from nvtabular import ColumnSchema, ColumnSelector, Schema, ops
+
+
+@pytest.mark.parametrize("properties", [{}, {"p1": "1"}])
+@pytest.mark.parametrize("tags", [[], ["TAG1", "TAG2"]])
+@pytest.mark.parametrize(
+    "op",
+    [
+        (ops.Bucketize, [1]),
+        (ops.Rename, {"postfix": "_trim"}),
+        ops.Categorify,
+        (ops.Categorify, {"encode_type": "combo"}),
+        (ops.Clip, 0),
+        (ops.DifferenceLag, "1"),
+        ops.FillMissing,
+        (ops.Groupby, ["1"]),
+        (ops.HashBucket, 1),
+        (ops.HashedCross, 1),
+        (ops.JoinGroupby, ["1"]),
+        (ops.ListSlice, 0),
+        ops.LogOp,
+        ops.Normalize,
+        (ops.TargetEncoding, ["1"]),
+    ],
+)
+@pytest.mark.parametrize("selection", [["1"], ["2", "3"], ["1", "2", "3", "4"]])
+def test_schema_out(tags, properties, selection, op):
+    # Create columnSchemas
+    column_schemas = []
+    all_cols = []
+    if isinstance(op, tuple):
+        if isinstance(op[1], dict):
+            op = op[0](**op[1])
+        else:
+            op = op[0](op[1])
+    else:
+        op = op()
+    for x in range(5):
+        all_cols.append(str(x))
+        column_schemas.append(ColumnSchema(str(x), tags=tags, properties=properties))
+    # Turn to Schema
+    schema = Schema(column_schemas)
+    # run schema through op
+    selector = ColumnSelector(selection)
+    new_schema = op.compute_output_schema(schema, selector)
+    # check schema otherside
```

Comment:
Add a blank line before the comments so my old eyes can parse this code into meaningful chunks to read and understand, huh? 😄